memote/utils.py

```diff
@@ -236,3 +236,18 @@ def jsonify(obj, pretty=False):
     except TypeError as error:
         log_json_incompatible_types(obj)
         raise_with_traceback(error)
+
+
+def flatten(list_of_lists):
```

Comment:
I think we might change the annotation object to always contain lists in future.

Reply:
Sounds good. Until we do we should just go with this workaround.